tests/debugger/utils.py

```diff
@@ -384,6 +389,16 @@ def _get_spans_hash(self):
 
         self.probe_spans = _get_spans_hash(self)
 
+    def _collect_symbols(self):
+        raw_data = list(interfaces.library.get_data(_SYMBOLS_PATH))
+
+        for data in raw_data:
+            if isinstance(data, dict) and "request" in data:
+                content = data["request"].get("content", [])
+                for part in content:
+                    if isinstance(part, dict) and "system-tests-file-path" in part:
+                        self.symbols.append(part["system-tests-file-path"])
```

Comment:
There is a small issue here : if this function is called N times, the list of symbols will have N duplicated values, as `symbols` is defined at class level (it will be the same list instance for all class instances).

You can simply change it to : 

```python
def _collect_symbols(self) -> list:
    result = []

    ...

    return result
```

Reply:
Thank you for bringing this to my attention. Fixed.